services/src/main/java/org/keycloak/authentication/requiredactions/ConsoleVerifyEmail.java

```diff
@@ -92,6 +93,8 @@ public void processAction(RequiredActionContext context) {
             event.error(Errors.INVALID_CODE);
             return;
         }
+
+        user.setLastUpdatedTimestampToNow();
```

Comment:
I'm not sure about the method name... what if we'd need to run additional code on user update?
How about "markAsUpdated()" instead of "setLastUpdatedTimestampToNow"?

Reply:
That's a good point. Didn't consider that... I will adjust this.

Reply:
@thomasdarimont: I've updated the PR accordingly.